doc/source/wellington_govt_nz.md

```diff
@@ -0,0 +1,50 @@
+# Wellington City Council
+
+Support for schedules provided by [Wellington City Council](https://wellington.govt.nz/).
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: wellington_govt_nz
+      args:
+        streetName: Street Name, Suburb # see 'How to get the source argument below'
+        streetId: streetID Number 
+```
+
+### Configuration Variables
+
+**streetName**<br>
+*(string)*<br>
+**streetId**<br>
+*(string)*<br>
+*One of the above is Required, you could use both, but not recommeded*
```

Comment:
Remove "but not recommended" because it is confusing.

Reply:
👍